src/index.html

```diff
@@ -7,8 +7,96 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <header class="header">
+    <div class="conteiner">
+      <div class="img">
+        <a
+          href="#home"
+          class="a-image"
+        ><img
+           src="images/logo.png"
+           alt="moyo"
+       ></a>
+      </div>
+      <nav class="child">
+        <ul class="text">
+          <a
+            href="#APPLE"
+            class="is-active">
+            APPLE
+          </a>
+        </ul>
```

Comment:
```suggestion
        <ul class="text">
          <a
            href="#APPLE"
            class="is-active"
          >
            APPLE
          </a>
        </ul>
```
format your code style and it should be `ul` container for whole list and every list element (link) should be wrapped into `li` tag